public/app/plugins/panel/geomap/layers/data/routeLayer.tsx

```diff
@@ -190,15 +197,26 @@ export const routeLayer: MapLayerRegistryItem<RouteConfig> = {
 
     // Crosshair layer
     const crosshairFeature = new Feature({});
-    const crosshairRadius = (style.base.lineWidth || 6) + 2;
+    const hLineFeature = new Feature({});
+    const vLineFeature = new Feature({});
+    const lineFeatures = [hLineFeature, vLineFeature];
+    const crosshairRadius = (style.base.lineWidth || 6) + 3;
     const crosshairStyle = new Style({
       image: new Circle({
         radius: crosshairRadius,
         stroke: new Stroke({
-          color: alpha(style.base.color, 0.4),
-          width: crosshairRadius + 2,
+          color: alpha('#607D8B', 1),
+          width: 1,
         }),
-        fill: new Fill({ color: style.base.color }),
+        fill: new Fill({ color: alpha('#607D8B', 0.4) }),
+      }),
+    });
+    const lineStyle = new Style({
+      stroke: new Stroke({
+        color: '#607D8B',
```

Comment:
should we split this color out into a reusable const as we are referencing it multiple time? 🤔